docker-compose.yml

```
@@ -374,3 +376,11 @@ services:
     image: ghcr.io/iden3/driver-did-iden3:v0.0.4
     ports:
       - "8152:8080"
+  
+  driver-did-earthid:
+    image: vibhi09/driver-did-earthid:latest
+    platform: linux/arm64/v8
+    environment:
+      baseUrl: "https://did.myearth.id/v1/resolve"
+    ports:
+      - "8153:8080"
```

I'm using 8153 for the driver-did-prism

```suggestion
      - "8154:8080"
```

---

I've updated the port number as suggested. Please let me know if any changes.